apps/cli/src/commands/run.ts

```diff
@@ -108,6 +108,11 @@ export default class Run extends Command {
         ];
         const attachment = flags.verbose ? [] : ["--attach", "validator"];
 
+        // create en empty .sunodo.env file if it doesn't exists
+        if (!fs.existsSync(".sunodo.env")) {
+            fs.writeFileSync(".sunodo.env", "");
```

Comment:
research a way to not need this

Reply:
already did, `compose` fails if the file doesn't exists

what I can do is create a `docker-compose-envfile.yaml` that would be added to `compose --file` only if the file exists.